scalyr_agent/configuration.py

```diff
@@ -417,6 +417,10 @@ def profile_clock( self ):
     def profile_log_name( self ):
         return self.__get_config().get_string( 'profile_log_name' )
 
+    @property
+    def disable_new_addevents_format( self ):
```

Comment:
Calling things "new_addvents_format" is not a great practice since, when there is another iteration of the `AddEvents`, we will be confused what "new" version this is referring to.

Let's call this `disable_logfile_addevents_format`.  It's a clunky name, but we shouldn't need to use it.. and we will delete it in a month otherwise.  You could argue that since there will not be another `new` format in the next month, it's not necessary.. but I'd like us to develop good habits.